content-src/components/TopSites/TopSiteFormInput.jsx

```diff
@@ -42,12 +52,14 @@ export class TopSiteFormInput extends React.PureComponent {
       <div className={`field ${typeUrl ? "url" : ""}${validationError ? " invalid" : ""}`}>
         {this.props.loading ?
           <div className="loading-container"><div className="loading-animation" /></div> :
-          showClearButton && <div className="icon icon-clear-input" onClick={this.props.onClear} />}
+          showClearButton && <div className="icon icon-clear-input" onClick={this.props.onClear} onKeyPress={this.onClearIconPress} role="button" tabIndex="0" />}
```

Comment:
I think it's a bit weird that this gets tabbed to first before the input itself, could you maybe just switch this to a button element instead?

Reply:
sure!